packages/terra-slider/src/Slider.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+  * Slider label value.
+  */
+  labelText: PropTypes.string.isRequired,
```

Comment:
Should this just be called `label` since "text" would be implied?

Reply:
Updated.